base/bucket_gocb.go

```diff
@@ -1236,6 +1244,33 @@ func (bucket *CouchbaseBucketGoCB) UpdateXattr(k string, xattrKey string, exp ui
 	err, cas = RetryLoopCas("UpdateXattr", worker, bucket.Spec.RetrySleeper())
 	if err != nil {
 		err = pkgerrors.Wrapf(err, "Error during UpdateXattr with key %v", UD(k).Redact())
+		return cas, err
+	}
+
+	if isDelete && !supportsTombstoneCreation {
+		worker := func() (shouldRetry bool, err error, value interface{}) {
+			builder := bucket.Bucket.MutateInEx(k, gocb.SubdocDocFlagNone, gocb.Cas(cas), exp).RemoveEx("", gocb.SubdocFlagNone)
```

Comment:
As discussed, want to set the two macro expansion values (cas and crc32c) as part of this mutation.  crc32c should just be DeleteCrc32c as on line 1225, cas is just the usual (line 1220).

It would be a good idea to have unit tests that verify that import isn't happening after these mutations occur.